staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1/conversion_test.go

```diff
@@ -605,3 +606,57 @@ func TestJSONConversion(t *testing.T) {
 		}
 	}
 }
+
+func TestJSONRoundTrip(t *testing.T) {
+	testcases := []struct {
+		name string
+		in   string
+		out  string
+	}{
+		{
+			name: "nulls",
+			in:   `{"default":null,"enum":null,"example":null}`,
+			out:  `{}`,
+		},
+		{
+			name: "null values",
+			in:   `{"default":{"test":null},"enum":[null],"example":{"test":null}}`,
+			out:  `{"default":{"test":null},"enum":[null],"example":{"test":null}}`,
```

Comment:
We need to reject it if people are trying to make `null` an enum option. That cannot work, it will break assumptions that `null` means you are trying to clear the field. I can't tell if that is what is going on here or not.

Reply:
a nullable value you want to constrain to a specific set of values or null must be able to include null as an enum value

Reply:
A field being set to null is not a condition we can express, e.g. it can't round-trip. Null is an enum value the way that bald is a hair color. I would not expect to have to list null as a possible value in order to use `field: null` to delete the field.

Reply:
it can/does round-trip in custom resources...

Reply:
OK I realize what is going on here now, I agree raw JSON should work like this, given that we have it at all, which I'm not super happy about...